geniza/corpus/templates/corpus/snippets/document_transcription.html

```diff
@@ -47,18 +47,20 @@
                 <div class="img" id="{{ image_info.label }}" data-controller="iiif" data-iiif-target="imageContainer">
```

Comment:
can we / should we omit the `data-controller` for images that are not part of the current document?